src/style.css

```diff
@@ -1 +1,86 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  align-self: center;
+  height: 40px;
+}
+
+.nav {
+  box-sizing: border-box;
+}
+
+.logo__img {
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  margin: 0 5px;
+  padding: 0 5px;
+  position: relative;
+}
+
+.nav__link {
+  height: 60px;
+  line-height: 60px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  pointer-events: all;
+  margin: 0;
+  padding: 0;
```

Comment:
your `.nav__link` is an `a` element. these do not have any default margins or paddings